Bottom: 1fc7c8175b5073b590b354c67807b0a5d0bf345f
Top:    c7bb300f633f1bf185faa67152a73135d69739e8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-24 13:38:28 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
